app/processing/payment/get-payment-schedule.js

```diff
@@ -0,0 +1,20 @@
+const moment = require('moment')
+const { DAX_CODES } = require('../../constants/schedules')
+const getSchedule = require('./get-schedule')
+
+const getPaymentSchedule = (schedule, dueDate, paymentValue, settledValue, previousSettledValue, totalValue, currentDate) => {
+  const scheduleDate = moment(dueDate, 'DD/MM/YYYY')
+
+  switch (schedule) {
+    case DAX_CODES.QUARTERLY:
+      return getSchedule(scheduleDate, 4, paymentValue, settledValue, previousSettledValue, totalValue, 3, 'month', currentDate)
+    case DAX_CODES.MONTHLY:
+      return getSchedule(scheduleDate, 12, paymentValue, settledValue, previousSettledValue, totalValue, 1, 'month', currentDate)
+    case DAX_CODES.THREE_DAY_QUARTERLY:
+      return getSchedule(scheduleDate, 4, paymentValue, settledValue, previousSettledValue, totalValue, 3, 'day', currentDate)
+    default:
+      throw new Error(`Unknown schedule ${schedule}`)
+  }
+}
```

Comment:
This is essentially the same pattern used in `ffc-pay-processing` to calculate the payment dates and outstanding status to prevent balloon reductions.  We can use similar logic to identify what is possible to be paid at each point in time and how far we have progressed through that schedule based on the settlement value.